### ISSUE TITLE
508-defect-1 [AXE-CORE]: COVID Vaccine Trials - DD and DT elements must be contained by a DL

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#18223]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I want to hear an accurate count of items in the description list on the form review page.

## Details

We have some DD and DT elements that are wrapped in two containing DIVs on the review page of our form, and it's throwing an axe-core automated violation. Removing the extra wrapper DIV should fix the issue. Screenshot and code snippet attached.

## Acceptance Criteria
- [ ] Visual styling is maintained
- [ ] Zero DD and DT elements must be contained by a DL violations reported on future axe-core scans

## Environment

* https://staging.va.gov/coronavirus-research/volunteer/review-and-submit

## Proposed Solution (if known)
We should remove the wrapper DIV. Maybe return a `React.Fragment` instead?

```html
- <div>
  <div class="review-row">
    <dt>Diagnosed by:</dt>
    <dd>With a positive antibody blood test</dd>
  </div>
- </div>
```

## WCAG or Vendor Guidance (optional)

* [Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)

## Screenshots or Trace Logs

![Screen Shot 2021-01-06 at 4 51 14 PM](https://user-images.githubusercontent.com/934879/103829484-86862780-5040-11eb-8066-6f47f6077ea7.png)

**Comment by stephenBarrs** [comment]
@1Copenut here is a PR with first pass at this fix.  Are you able to test this branch and let me know if it resolves the issue?  
https://github.com/department-of-veterans-affairs/vets-website/pull/15614

**Comment by 1Copenut** [comment]
@stephenBarrs Yep, I'll give it a look before EOD. I'm prepping a few things for a meeting in 30 atm.

**Comment by 1Copenut** [comment]
Just re-checked in Staging and axe-core violations are gone. Moving to close.
